src/compressed_tensors/transforms/transform_config.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict
+
+from compressed_tensors.transforms.transform_scheme import TransformationScheme
+from pydantic import BaseModel
+
+
+__all__ = ["TransformationConfig"]
+
+
+class TransformationConfig(BaseModel):
```

Comment:
We should probably record `TransformationScheme`s within `QuantizationConfig.config_groups`

```python3
class QuantizationConfig:
    config_groups: Dict[str, Union[QuantizationScheme, TransformationScheme, List[str]]]
```

Especially when we add online transformations, we'll want a way of serializing this transformation information

Reply:
Layers being quantized may not all have transforms and layer with transforms may not be quantized

Reply:
@dsikka See updated comment. I submitted my review too early, sorry about the confusion

Reply:
It is within the QuantizationConfig and we do get serialization (not just for online rotations, but for cases where we are not saving compressed).

E.g: `nm-testing/Llama-3.2-1B-Instruct-W4A16-uncompressed-mse-hadamard`